### PR TITLE
[#150373524] make 'created' optional

### PR DESCRIPTION
**Pull request by MothOnMars**

### CODE REVIEW

File: spec/requests/api/v1/documents_spec.rb [hunk 1]
```diff
@@ -3,6 +3,13 @@
 
 describe API::V1::Documents do
   let(:id) { "some really/weird@id.name" }
+  let(:valid_params) do
+    {
+     "document_id" => id, "title" => "my title", "path" => "http://www.gov.gov/goo.html",
```
Comment (noremmie) [inline spec/requests/api/v1/documents_spec.rb]: nit: I would have expected this to be indented by one more space.

Comment (MothOnMars) [inline spec/requests/api/v1/documents_spec.rb]: Can do. For some reason I thought this was correct, but the style guide agrees with you, so...

**Comment by MothOnMars** [comment]
Slapping a DNM label on this until https://github.com/GSA/usasearch/pull/354 is merged.

File: spec/requests/api/v1/documents_spec.rb [hunk 2]
```diff
@@ -5,9 +5,14 @@
   let(:id) { "some really/weird@id.name" }
   let(:valid_params) do
     {
-     "document_id" => id, "title" => "my title", "path" => "http://www.gov.gov/goo.html",
-     "description" => "my desc", "promote" => true,
-     "language" => 'hy', "content" => "my content", "tags" => "Foo, Bar blat",
+      "document_id" => id,
```
Comment (noremmie) [inline spec/requests/api/v1/documents_spec.rb]: I wasn't going to say anything about this, and I'm glad you flattened this out :)

**Comment by noremmie** [comment]
This all still LGTM 👍

**Pull request merged**